index.html

```diff
@@ -4,6 +4,51 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Healthy Resturant</title>
+    <link rel="stylesheet" href="styles.css" />
   </head>
-  <body></body>
+  <body>
+    <header>
+      <div class="logo">
+        <img src="/Images/logo 1.svg" alt="" width="24px" height="24px" />
+        <p>Tasty</p>
+      </div>
+      <div>
+        <nav>
+          <div>Home</div>
+          <div>About</div>
+          <div>Menu</div>
+          <div>New</div>
+          <input class="support-button" type="button" value="Support" />
+        </nav>
+      </div>
+    </header>
+    <main>
+      <div class="hero-section">
+        <div class="left">
+          <img src="/Images/home1-img 1.svg" alt="" />
+          <div class="dishname">
+            <p>
+              Fish Salad <br />
+              Fish Salad with Sweet <br />
+              Potatoes and Basil
+            </p>
+          </div>
+        </div>
+        <div class="right">
+          <div class="heading1"><p>#1 OF THIS WEEK</p></div>
+          <div class="heading2">
```

Comment:
heading1, heading2 are not good examples of classnames.
Can you make them more sensible??

Reply:
done